cmd/pulumi-test-language/interface.go

```diff
@@ -405,9 +405,7 @@ func (h *testHost) CloseProvider(provider plugin.Provider) error {
 	return nil
 }
 
-func (h *testHost) LanguageRuntime(
-	root, pwd, runtime string, options map[string]interface{},
-) (plugin.LanguageRuntime, error) {
+func (h *testHost) LanguageRuntime(runtime string, info plugin.ProgramInfo) (plugin.LanguageRuntime, error) {
```

Comment:
We should leave a comment here that ProgramInfo is only being passed in for compat reasons and should be removed at some point.

Reply:
Done ✅